recipes.adoc

```diff
@@ -11,34 +11,11 @@ and BRS-B (for "Bespoke").
 
 === BRS-I Recipe
 
-The BRS-I recipe aims to simplify end-user experiences, software
-compatibility and OS distribution, by definining a common specification
-for boot and runtime interfaces. BRS-I is expected to be used by
-general-purpose compute devices such as servers, desktops, laptops and
-other devices with industry expectations on silicon vendor, OS and software
-ecosystem interoperatbility. BRS-I enables operating system
-providers to build a single *generic* operating system image that can be
-*successfully booted* on compliant systems. *Generic* means not requiring
-system-specific customizations - only an implementation of BRS-I
-requirements. *Successfully boot* means basic system configuration,
-sufficient for detecting the need for system-specific drivers and
-loading such drivers.
+BRS-I is expected to be used by general-purpose compute devices such
+as servers, desktops, laptops and other devices with industry
+expectations on silicon vendor, OS and software ecosystem
+interoperatbility.
 
-It is understood that systems will deliver features beyond those covered
```

Comment:
This specific section was added because it is not covered elsewhere, and does not apply to BRS-B systems at all.

Reply:
The notion of being able to boot "generic OS" images with no ill side effects is one that makes sense for server, PC, laptop and other horizontal markets (e.g. BRS-I).

Reply:
It also makes sense for for a BRS-B system to be able to do the same.  It also makes sense for any other BRS recipe system to do likewise.  This is described in the introduction to the BRS Specification.

Reply:
Actually, yes, the goal of interoperability is stated in the very first sentence of the BRS Specification, in the Introduction:

> The Boot and Runtime Services (BRS) specification provides the software requirements for system vendors and Operating System Vendors (OSVs) to **interoperate** with one another by providing expectations for the Operating System (OS) to utilize in acts of device discovery, system management, and other rich operations provided in this specification.

This applies to the BRS Specification as a whole as well as to all recipes.

Just because the BRS-B isn't concerned with **Very Important Commercial Operating Systems®™**, doesn't mean that interoperability is irrelevant.  The BRS-B, and presumably other future recipes, are concerned with interoperability between different sets of implementations and operating systems.

If you feel that the term "bespoke" strongly indicates a lack of interoperablity, then perhaps that receipe needs a different, more appropriate name.